googleMapsApiModule.py

```diff
@@ -20,15 +23,29 @@ def direction(origin,destination):
     logging.debug(result_url)
     webbrowser.open_new(result_url)
 
-def timezone(search_location,timestamp):
-    address = search_location.lower()
-    address = address.replace(" ","+")
-    result_url = f'https://maps.googleapis.com/maps/api/timezone/json?location={address}&timestamp={timestamp}&key={config.key}
-    print(result_url)
-    webbrowser.open_new(result_url)
+def get_timestamp(datetime):        #format of datetime should be YYYY MM DD Hrs Mins
+	yr,mon,day,hr,mi=map(int,datetime.split())
+	d=datetime(yr,mon,day,hr,mi)
+	timestamp = calendar.timegm(d.timetuple())
+	return timestamp
+
```

Comment:
Add `@logger_config.logger` before each new method.

Reply:
@vishakha-lall I have made the necessary changes suggested by you.